README.md

```diff
@@ -143,9 +146,17 @@ Make copies of the [example manifests](./manifests/example) for editing. The nec
 
 1. `metadata.name:` Arbitrary name of the secret. Must match the PVC's `cdi.kubevirt.io/storage.import.secretName:`
 
-1.  `accessKeyId:` Contains the endpoint's key and/or user name. This value **must be base64 encoded** with no extraneous linefeeds. Use `echo -n "xyzzy" | base64` or `printf "xyzzy" | base64` to avoid a trailing linefeed
+1.  `data.accessKeyId:` Contains the endpoint's key and/or user name. This value **must be base64 encoded** with no extraneous linefeeds. Use `echo -n "xyzzy" | base64` or `printf "xyzzy" | base64` to avoid a trailing linefeed
```

Comment:
be aware that this is changing the api for anyone who has build logic around how secrets are constructed.

Reply:
Actually we didn't changed the api, just altered the readme. I.e. accessKeyId and secretKey are already under 'data' (see secret yaml[1]). So we've just added 'stringData' and updated the readme accordingly.

[1] https://github.com/kubevirt/containerized-data-importer/blob/master/manifests/example/endpoint-secret.yaml